package.json

```diff
@@ -5,9 +5,9 @@
   "main": ".eslintrc.js",
   "scripts": {
     "dev": "./node_modules/.bin/nodemon index.js",
-    "lint": "run lint:pug-lint && run lint:eslint",
+    "lint": "run lint:puglint && run lint:eslint",
```

Comment:
`"lint": "npm run lint:puglint && npm run lint:eslint",` Someone didn't use this command :trollface: